protocols/epic.js

```diff
@@ -15,20 +15,28 @@ export default class Epic extends Core {
     this.clientSecret = null
     this.deploymentId = null
     this.epicApi = 'https://api.epicgames.dev'
+    this.alternativeAuth = false // Some games require a client access token to POST to the matchmaking endpoint.
```

Comment:
`alternativeAuth` doesnt sounds very indicative of what it is, this could be renamed to something like `authByExternalToken` (or something else).